src/pages/Host.tsx

```diff
@@ -1,8 +1,36 @@
 import React from "react";
 import { useParams } from "react-router-dom";
+import { GET_HOST } from "gql/queries/get-host";
+import get from "lodash/get";
+import { useQuery } from "@apollo/react-hooks";
+import { HostQuery, HostQueryVariables } from "gql/generated/types";
+import { usePageTitle } from "hooks/usePageTitle";
+import { PageWrapper } from "components/styles";
+import { HostStatusBadge } from "components/HostStatusBadge";
+import { PageTitle } from "components/PageTitle";
+import { HostStatus } from "types/host";
 
 export const Host: React.FC = () => {
   const { id } = useParams<{ id: string }>();
+  // Query host data
+  const { data, loading } = useQuery<HostQuery, HostQueryVariables>(GET_HOST, {
+    variables: { id },
+  });
+  const host = get(data, "host");
```

Comment:
We started moving away from lodash.get and have adopted optional chaining so that we can preserve typescript typing.

you can do 
```
const host = data?.host;
const hostUrl = host?.hostUrl;
```